tests/foreman/ui/test_role.py

```diff
@@ -127,7 +127,7 @@ def test_positive_create_role_with_taxonomies(self):
 
         @id: 5d9da688-f371-4654-93d3-b221211be280
 
-        @steps: Create new role with taxonomies
+        @steps: Create new role with taxonomies(location and organization)
```

Comment:
nitpick: should there be a space after taxonomies? Also repeated several times below